library/src/scala/collection/concurrent/TrieMap.scala

```diff
@@ -1049,7 +1049,7 @@ object TrieMap extends MapFactory[TrieMap] {
   def newBuilder[K, V]: mutable.GrowableBuilder[(K, V), TrieMap[K, V]] = new GrowableBuilder(empty[K, V])
 
   @transient
-  val inodeupdater: AtomicReferenceFieldUpdater[INodeBase[_, _], MainNode[_, _]] = AtomicReferenceFieldUpdater.newUpdater(classOf[INodeBase[_, _]], classOf[MainNode[_, _]], "mainnode")
+  val inodeupdater: AtomicReferenceFieldUpdater[INodeBase[_, _], MainNode[_, _]] = INodeBase.updater
```

Comment:
Maybe `TrieMap.inodeupdater` is unused. but error without this change. Should we deprecate this field? 🤔 

```
Caused by: java.lang.RuntimeException: java.lang.IllegalAccessException: class scala.collection.concurrent.TrieMap$ cannot access a member of class scala.collection.concurrent.INodeBase with modifiers "private volatile"
	at java.base/java.util.concurrent.atomic.AtomicReferenceFieldUpdater$AtomicReferenceFieldUpdaterImpl.<init>(AtomicReferenceFieldUpdater.java:351)
	at java.base/java.util.concurrent.atomic.AtomicReferenceFieldUpdater.newUpdater(AtomicReferenceFieldUpdater.java:115)
	at scala.collection.concurrent.TrieMap$.<clinit>(TrieMap.scala:1052)
```